capstone-frontend/package.json

```diff
@@ -6,8 +6,12 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "acorn": "^7.3.1",
+    "parser": "^0.1.4",
```

Comment:
Hi Steven, what is the acorn and parser dependencies and what do they do?

Reply:
I npm installed these because I was getting errors that they were undefined, but upon removing them everything seems to be fine. I'll take them out.